protokube/pkg/protokube/kube_boot.go

```diff
@@ -154,8 +155,14 @@ func enableKubelet() error {
 	// TODO: Check/log status of kubelet
 	// (in particular, we want to avoid kubernetes/kubernetes#40123 )
 	glog.V(2).Infof("ensuring that kubelet systemd service is running")
-	cmd := exec.Command("systemctl", "start", "--no-block", "kubelet")
+	cmd := exec.Command("systemctl", "status", "--no-block", "kubelet")
 	output, err := cmd.CombinedOutput()
+	if err == nil {
+		glog.V(2).Infof("kubelet systemd service already running")
+		return nil
+	}
+	cmd = exec.Command("systemctl", "start", "--no-block", "kubelet")
```

Comment:
Yes I understand ;) What I am asking is should we log that kubelet is not running and we are starting it.